src/display/xfa_layer.js

```diff
@@ -30,13 +81,20 @@ class XfaLayer {
         Object.assign(html.style, value);
       }
     }
+
+    // Set the value after the others to be sure overwrite
+    // any other values.
+    if ("fieldId" in attributes && storage) {
+      XfaLayer.setupStorage(html, attributes.fieldId, element, storage);
```

Comment:
Nit: Here, and elsewhere in this file, note that `static` methods should be able to call each other directly; hence `this.setupStorage(...)` ought to work.